### PR TITLE
Logic develop

### PR DESCRIPTION
**Pull request by KoTuK2306**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,32 +10,37 @@
...
-        <button class="button">0</button>
-        <button class="button">,</button>
-        <button class="button">=</button>
+      <div class="resultWindow">
+        <p id="result">0</p>
+      </div>
+      <div id="buttonsBlock" class="buttons">
+        <button id="b1" class="button">%</button>
+        <button id="b2" class="button">CE</button>
+        <button id="b3"  class="button">C</button>
```
Comment (pavelsokolov21) [inline index.html]: extra space

File: script.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+let buttons = document.getElementById("buttonsBlock");
```
Comment (pavelsokolov21) [inline script.js]: const

File: script.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+let result = document.getElementById("result");
```
Comment (pavelsokolov21) [inline script.js]: const

File: script.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+buttons.onclick = function (event) {
+    if (event.target.id == "b3") {
```
Comment (pavelsokolov21) [inline script.js]: `id=b3` isn't clear. Use `data-*` attributes instead

**Review by pavelsokolov21 — CHANGES_REQUESTED** [review]

File: script.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+let buttons = document.getElementById("buttonsBlock");
+let result = document.getElementById("result");
+
+buttons.onclick = function (event) {
```
Comment (pavelsokolov21) [inline script.js]: `addEventListener` instead

File: script.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+    if (event.target.id == "b3") {
+        result.innerHTML = 0;
```
Comment (pavelsokolov21) [inline script.js]: `return;` as next line. Then `else` below will be unnecessary

File: styles.css [hunk 7]
```diff
@@ -25,11 +25,6 @@ body {
   margin: auto;
 }
 
-.result {
```
Comment (pavelsokolov21) [inline styles.css:28]: why?

File: script.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
+const buttons = document.getElementById("buttonsBlock");
```
Comment (pavelsokolov21) [inline script.js]: yYou can use getElementsByClassName. A lot of ids aren't a good idea.
```
const [buttons] = document.getElementsByClassName("buttonsBlock");
```

File: script.js [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+const result = document.getElementById("result");
+
+buttonClick = (event) => {
```
Comment (pavelsokolov21) [inline script.js]: Wow, you made the function without declaration (const or let). That's bad practice!

File: script.js [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+    if (event.target.id == "clean") {
```
Comment (pavelsokolov21) [inline script.js]: use the strong equal `===`

File: index.html [hunk 11]
```diff
@@ -10,32 +10,34 @@
...
-        <button class="button operation">-</button>
-        <button class="button">1</button>
-        <button class="button">2</button>
-        <button class="button">3</button>
-        <button class="button operation">+</button>
-        <button class="button">+/-</button>
-        <button class="button">0</button>
-        <button class="button">,</button>
-        <button class="button">=</button>
+      <div class="result">
```
Comment (pavelsokolov21) [inline index.html]: why not `p` directly? If you use it you won't have an id here

**Review by pavelsokolov21 — CHANGES_REQUESTED** [review]

File: script.js (hunk 9 above)
Comment (KoTuK2306) [reply script.js]: I forgot about this. Sorry

File: index.html (hunk 11 above)
Comment (pavelsokolov21) [inline index.html]:
```
<p class="result"></p>
```

File: script.js [hunk 12]
```diff
@@ -0,0 +1,12 @@
+const buttons = document.getElementsByClassName("buttonsBlock");
```
Comment (pavelsokolov21) [inline script.js]: it won't work. getElementsByClassName returns an array of elements. Use destruction`[buttons]`

File: script.js [hunk 13]
```diff
@@ -0,0 +1,12 @@
...
+const result = document.getElementsByTagName("p");
```
Comment (pavelsokolov21) [inline script.js]: change results to `p` with a class and use `getElementsByClassName`

**Review by pavelsokolov21 — CHANGES_REQUESTED** [review]

File: script.js [hunk 14]
```diff
@@ -0,0 +1,12 @@
+const [buttonsContainer] = document.getElementsByClassName("buttonsBlock");
+const [result] = document.getElementsByClassName("result");
+
+let buttonClick = (event) => {
```
Comment (pavelsokolov21) [inline script.js]: const

File: index.html [hunk 15]
```diff
@@ -10,32 +10,32 @@
...
-        <button class="button">=</button>
+      <p class="result">0</p>
+      <div class="buttonsBlock">
+        <button id="percent" class="button operation">%</button>
+        <button id="deleteLastOperation" class="button operation">CE</button>
+        <button id="clean" class="button operation">C</button>
+        <button id="backspace" class="button operation">⌫</button>
+        <button id="partOfAWhole" class="button operation">1/x</button>
+        <button id="square" class="button operation">x<sup>2</sup></button>
+        <button id="root" class="button operation">√x</button>
```
Comment (pavelsokolov21) [inline index.html]: `sqrt`. Root means the main folder

File: index.html [hunk 16]
```diff
@@ -10,32 +10,32 @@
...
+        <button id="9" class="button">9</button>
+        <button id="multiple" class="button operation">x</button>
+        <button id="4" class="button">4</button>
+        <button id="5" class="button">5</button>
+        <button id="6" class="button">6</button>
+        <button id="minus" class="button operation">-</button>
+        <button id="1" class="button">1</button>
+        <button id="2" class="button">2</button>
+        <button id="3" class="button">3</button>
+        <button id="summ" class="button operation">+</button>
```
Comment (pavelsokolov21) [inline index.html]: `sum`

File: script.js [hunk 17]
```diff
@@ -0,0 +1,12 @@
...
+  if (event.target.id === "clean") {
+    result.innerHTML = 0;
+    return;
+  }
```
Comment (pavelsokolov21) [inline script.js:8]: I'd add space between `}` and the next line

**Review by pavelsokolov21 — CHANGES_REQUESTED** [review]

**Review by pavelsokolov21 — APPROVED** [review]

**Pull request merged**